SingleSource/UnitTests/AArch64/acle-fmv-features.c

```diff
@@ -363,15 +363,23 @@ CHECK(mops, mops, mops, {
      );
 })
 
-static bool safe_try_feature(bool (*try_feature)(void)) {
+bool (*whitelist[])(void) = { try_ssbs2 };
```

Comment:
suggest: s/whitelist/allowlist/, along with a comment explaining what the allow-list permits?